### PR TITLE
Add plug&play Dockerfile, fix imports, unpin requirements

### PR DESCRIPTION
**Pull request by ddelange**
This PR is basically a trial&error walk through your Installation wiki, accumulating all those steps into a ready-to-use Dockerfile (uses your db.sqlite3 file by default, although the ADMIN_PASSWORD=password `ENV` statement obviously won't work out of the box with your db). Please do squash if you like the PR, to get rid of all the dev commits 😅  the diff without whitespaces is about half the size.

- Add plug&play Dockerfile, the django app now respects DEBUG, DATABASE_URL and ADMIN_ env vars (in case the db doesn't have an admin yet).
- Fix relative and wildcard imports, also for Adaptive-NER methods that were previously still commented out. I renamed `from alpaca_serving.client import AlpacaClient` to `from alpaca_serving_client.client import AlpacaClient` due to python package namespace conflict with the `alpaca_serving` package when installed in the same env (the Dockerfile).
- Unpin requirements. With pips new resolver (20.3), lower pins are perfectly fine. The Dockerfile boots fine with django3, latest pytorch etc.
- Switch from glove to bert in `SequenceTaggingModel` to avoid (hardcoded) glove embedding asset download error (ref https://github.com/flairNLP/flair/issues/1831)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
+FROM python:3.6-slim-buster as final
```
Comment (ddelange) [inline Dockerfile:1]: as alternative base image, I considered [`pytorch/pytorch:1.6.0-cuda10.1-cudnn7-runtime`](https://hub.docker.com/r/pytorch/pytorch/tags?page=1&ordering=last_updated), which runs py3.7 and will optionally pick up GPUs with a `docker run --gpus all ...` if the host runner has the NVIDIA Container Toolkit [installed](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#setting-up-nvidia-container-toolkit).

**Comment by Rahul-Khanna** [comment]
hi @ddelange, I would be open to doing a call to understand all these changes. I'm decently familiar with this system, and am building another one based off this project (https://github.com/INK-USC/LEAN-LIFE), so would love to dockerize that project. Let me know if that works for you. I also can merge this request in once I understand what's going on.

**Comment by Rahul-Khanna** [comment]
thanks for doing this by the way!

**Comment by ddelange** [comment]
Assuming you're in CA: good morning @Rahul-Khanna! 

You can reach me at t.me/Rhiniodon (my tz is GMT+1 btw), I should be able to do a quick Zoom or so spontaneously.

**Comment by ddelange** [comment]
So with eee2c98, `alpaca-serving` started successfully responding to django. However, it seems that there are some front-end issues with the current codebase (probably somewhere [here](https://github.com/ddelange/AlpacaTag/blob/ddbf3caaa566ee11f297067025c2a1d115d96b96/annotation/AlpacaTag/server/static/js/sequence_labeling.js#L43-L64)), especially with the annotation itself (some offset of around 20 chars versus what is selected). My hope here is that somehow merging a (way) more recent version of doccano into the codebase would fix it. Do you have time/ideas how to go about that?

ezgif com-gif-maker (1)

**Comment by ddelange** [comment]
So looking at the `div`, there are actually exactly 20 leading spaces:

image


I tried finding them, thought it was these 20 spaces (but it wasnt) https://github.com/ddelange/AlpacaTag/blob/75f9723765ea2ba4deb296ef7cd0e7ef42278c7e/annotation/AlpacaTag/server/static/js/sequence_labeling.js#L25

and checked out the corresponding html but didn't find the cause.
https://github.com/ddelange/AlpacaTag/blob/75f9723765ea2ba4deb296ef7cd0e7ef42278c7e/annotation/AlpacaTag/server/templates/annotation/sequence_labeling.html#L29

**Comment by ddelange** [comment]
Alright. Tagging is working now! I wanted webpack (`npm run build`) to build the `bundle` folder on the fly (instead of having it in git), but webpack seems to not build the bundle folder so I resorted to removing the 20 spaces manually from the bundle js (a03a001) as well as from the source js (9893f53). This did the trick :)

**Comment by danny911kr** [comment]
Hi, @ddelange Thank you so much for doing this.
I just get involved in the review of your huge efforts!
I'll get back to you as soon as possible.

**Pull request merged**